lib/bolt/bolt_option_parser.rb

```diff
@@ -709,7 +709,7 @@ def initialize(options)
           File.expand_path(moduledir)
         end
       end
-      define('--boltdir FILEPATH',
+      define('--boltdir FILEPATH', '--project FILEPATH',
```

Comment:
If we're updating the documentation to reference `--project` over `--boltdir`, then I think `--project` should come first. Otherwise, the command reference doc does not show `--project`.

If we put `--project` first then it will need to be added to the `OPTIONS[:global_config_setters]` constant.

Reply:
https://puppet.com/docs/bolt/latest/bolt_command_reference.html I think it will show both, but putting it first can't hurt.